src/v1/bookgroup/BookGroupDetail.tsx

```diff
@@ -0,0 +1,249 @@
+import Image from 'next/image';
+
+import Badge from '@/ui/Base/Badge';
+import { IconCalendar, IconMembers, IconComments } from '@public/icons';
+
+interface BookGroupDetailProps {
+  title: string;
+  description: string;
+  book: { title: string; bookImageSrc: string };
+  date: { start: string; end: string };
+  owner: { name: string; profileImageSrc: string };
+  memberCount: number;
+  commentCount: number;
+  isPublic: boolean;
+  handleClick: () => void;
+}
+
+type BookGroupStatus = 'before' | 'dday' | 'ongoing' | 'end';
+
+const getBookGroupStatus = (ddayByStart: number, ddayByEnd: number) => {
+  if (ddayByStart > 0) {
+    return {
+      status: 'before' as const,
+      ddayCount: ddayByStart,
+    };
+  } else if (ddayByStart === 0 && ddayByEnd > 0) {
+    return {
+      status: 'dday' as const,
+    };
+  } else if (ddayByStart < 0 && ddayByEnd >= 0) {
+    return {
+      status: 'ongoing' as const,
+    };
+  } else {
+    return {
+      status: 'end' as const,
+    };
+  }
+};
+
+const toDayFromMillseconds = (value: number) => {
+  return Math.ceil(value / (1000 * 60 * 60 * 24));
+};
+
+const BookGroupDetail = ({
+  title,
+  description,
+  book,
+  date,
+  owner,
+  memberCount,
+  commentCount,
+  isPublic,
+  handleClick,
+}: BookGroupDetailProps) => {
+  const ddayByStart = toDayFromMillseconds(
+    new Date(date.start).getTime() - new Date().getTime()
+  );
+  const ddayByEnd = toDayFromMillseconds(
+    new Date(date.end).getTime() - new Date().getTime()
+  );
+
+  const { ...ddayProps } = getBookGroupStatus(ddayByStart, ddayByEnd);
+
+  return (
+    <div
+      onClick={handleClick}
+      className="h-[16.142rem] w-[35.5rem] rounded-[0.4rem] px-[1.6rem] py-[0.9rem] shadow-[0_0_0.6rem_rgba(180,180,180,0.25)]"
+    >
+      <div className="mb-[1rem] flex gap-[0.5rem]">
+        <Dday {...ddayProps}></Dday>
+        <Public isPublic={isPublic}></Public>
```

Comment:
큰 차이는 없겠지만, 개인적으로는 children을 전달하지 않는 컴포넌트들은 self-closing 해주면 더 보기 좋은 것 같아요! 😀

```suggestion
        <Dday {...ddayProps} />
        <Public isPublic={isPublic} />
```